browser/ui/webui/brave_untrusted_web_ui_controller_factory.cc

```diff
@@ -21,13 +21,12 @@ namespace {
 
 WebUIConfigList CreateConfigs() {
   WebUIConfigList config_list;
-  auto register_config =
+  [[maybe_unused]] auto register_config =
```

Comment:
not sure I understand this one, `register_config` does get used, right?

Reply:
Looks like the original use of `ALLOW_UNUSED_LOCAL` was not needed to begin with, fixed in https://github.com/brave/brave-core/commit/9fd1d7539507d7a0d8fc6a1c3e9abaaaba92aeff